src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -370,7 +396,7 @@ pub struct NodeModel {
 
 impl NodeModel {
     /// Constructor.
-    pub fn new(app:&Application, registry:visualization::Registry) -> Self {
+    pub fn new(app:&Application,registry:visualization::Registry) -> Self {
```

Comment:
```suggestion
    pub fn new(app:&Application, registry:visualization::Registry) -> Self {
```
we do put space in argument list (to clearly separate arguments).